src/components/plots/ElbowPlot.jsx

```diff
@@ -131,8 +131,8 @@ ElbowPlot.defaultProps = {
 ElbowPlot.propTypes = {
   stylingConfig: PropTypes.object.isRequired,
   calculationConfig: PropTypes.object.isRequired,
-  plotData: PropTypes.object.isRequired,
-  actions: PropTypes.object.isRequired,
+  plotData: PropTypes.array.isRequired,
+  actions: PropTypes.bool.isRequired,
```

Comment:
Actions can actually be boolean or object : `https://github.com/vega/vega-embed`. Maybe you can use `PropTypes.oneOfType`?

```
  // An object that could be one of many types
  optionalUnion: PropTypes.oneOfType([
    PropTypes.bool,
    PropTypes.object,
  ])
```

Reply:
yup, that works too. I did not see the object type used anywhere in the code though

Reply:
Yup, I don't think it's currently used yet.